modules/ROOT/pages/database-administration/standard-databases/naming-databases.adoc

```diff
@@ -20,9 +20,7 @@ Support for database names starting with a numeric character is available from N
 The `-` (dash) and `.` (dot) characters are not legal in Cypher variables.
 Names containing a `-` or that begin with a numeric character must be enclosed within backticks.
 For example, `CREATE DATABASE ++`main-db`++` is a valid database name.
-Database names are the only identifier for which dots do not need to be quoted.
-For example `main.db` is a valid database name.
```

Comment:
wouldn't these two make sense to keep? 🤔

Reply:
I'm fine with it, but I still think having `Database names are the only identifier for which dots do not need to be quoted.` might be nice, since that is something that separates the database names with other identifiers 🤷